.github/ISSUE_TEMPLATE/update_truth.md

```diff
@@ -0,0 +1,60 @@
+---
+name: Update Truth
+about: Review use case differences that are caused by changes in an external repository and update truth dataset if necessary.
+title: 'Update Truth: '
+labels: 'alert: NEED ACCOUNT KEY, alert: NEED MORE DEFINITION, alert: NEED CYCLE ASSIGNMENT, type: update truth, priority: blocker, component: CI/CD'
```

Comment:
```suggestion
labels: 'alert: NEED ACCOUNT KEY, alert: NEED MORE DEFINITION, alert: NEED CYCLE ASSIGNMENT, type: update truth, priority: blocker, component: CI/CD, requestor: METplus Team'
```
Wondering if we should remove `alert: NEED ACCOUNT KEY` and `alert: NEED MORE DEFINITION` since this already pretty specific and adding an account key might be overkill. Not sure on that.

But we might as well also add `requestor: METplus Team` to keep the labeling easy.

Reply:
I think the account key would likely be the same key for the issue that warranted the Update Truth issue, but the label serves to note that it hasn't been added yet.

Similarly the `alert: NEEDS MORE DEFINITION` label could useful if the issue was created but no text was added to the body. Someone may create the issue to avoid forgetting but need to fill in the PR and issue numbers later. Maybe I'm overthinking it...